provision-osx/README.md

```diff
@@ -12,151 +12,145 @@ Provision Jenkins node for the Digger. This role performs the following tasks:
 * [Confifure Proxy](#configure-proxy)
 
 ## Prerequisites
-* SSH access as a user with `sudo` permissions. If a password is required, enter it in the `ansible_become_pass` variable.
-* //only way how to enter a pass?
-* `ansible_sudo_pass` variable set when running the job.
+* SSH access as a user with `sudo` permissions. If a password is required, declare it in the  `ansible_become_pass` variable.
 
-* `xcode_install_user` variable set when running the job, otherwise Xcode tasks will be skipped.
-* `xcode_install_password` variable set when running the job, otherwise Xcode tasks will be skipped.
+* `ansible_sudo_pass` variable set
 
-## Required Network Access
-The job supports the use of a proxy.
-
-<aside class="warning">
-Only proxies that do not require authentication are supported.
-</aside>
+* `xcode_install_user` and `xcode_install_password` variables set, otherwise Xcode tasks will be skipped.
 
+## Required Network Access
 
 The node will need access to the following external hosts:
-* `https://github.com` - To download and install github and Cocoapods.
+* `https://github.com` - To download and install github(???) and Cocoapods.
 * `https://raw.githubusercontent.com` - To download and install NVM.
 * `https://rvm.io` - To download and install RVM.
 * `http://developer.apple.com` - To download developer certificates and Xcode.
 * `https://npmjs.org` - To install NPM packages.
 
 ***Note: Other external hosts may be required depending on what other packages you specify to install.***
 
-## Install Homebrew
-Installs Homebrew on the node.
+## Install Homebrew Package Manager
+To run this section as a standalone step, use `osx_install_homebrew` tag in the playbook.
 
-This will also add any taps and packages specified in the options.
+This will also add "taps" and packages specified in the options.
 
-To run this section as a standalone step you must specify the `osx_install_homebrew` tag.
 
-### Options
+### Variables
 * `homebrew_version` - The version of Homebrew to install (git tag).
 * `homebrew_packages` - The packages to install using Homebrew. Format: `{ name: <PACKAGE_NAME> }`.
 * `homebrew_repo` - The git repo where Homebrew resides (defaults to GitHub repo).
-* `homebrew_prefix` -
+* `homebrew_prefix` - (???)
```

Comment:
what is this?

Reply:
Its a prefix used for homebrew_install_path (next line), not sure how to describe it.

Reply:
hm, i think it is low priority then, ill let it pass